openedx/core/djangoapps/content/learning_sequences/models.py

```diff
@@ -67,6 +67,22 @@ class Meta:
         ]
 
 
+class CourseContext(TimeStampedModel):
+    """
+    A model containing course specific information e.g course_visibility
+    """
+    COURSE_VISIBILITIES = [
+        ("private", "private"),
+        ("public_outline", "public_outline"),
+        ("public", "public"),
+    ]
```

Comment:
Can you derive this from the enum so they can't fall out of sync?

Reply:
@ormsbee Done. But, I had to empty the https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/content/learning_sequences/api/__init__.py file as it was causing circular import when I tried to import `api.data` from models

Reply:
Ah. Hmph. I do like being able to import from the top level of `learning_sequences.api` package... how about if we move `data.py` out of the `api` package and move it to the top level to make it a peer of `views.py` and `models.py`?

Reply:
@ormsbee Done. It solves the issue